src/connections/accounts.js

```diff
@@ -0,0 +1,66 @@
+import { withMutations } from 'cozy-client'
+
+const ACCOUNTS_DOCTYPE = 'io.cozy.accounts'
+const PERMISSIONS_DOCTYPE = 'io.cozy.permissions'
+
+const withAccountsMutations = withMutations(client => {
+  const createAccount = async (attributes, relationships) => {
+    const { data } = await client.create(
+      ACCOUNTS_DOCTYPE,
+      attributes,
+      relationships
+    )
+    return data
+  }
+
+  const findMasterAccount = async id => {
+    const { data } = await client.query(
+      client.find(ACCOUNTS_DOCTYPE).where({
+        _id: id
+      })
+    )
+    return data[0]
```

Comment:
je m'incruste vite fait :D 

tu dois pouvoir faire un ```
const {data } = await client.query(client.get(ACCOUNTS_DOCTYPE, id));
return data
```

Reply:
@Crash-- Le comportement est finalement un poil différent, `get` retourne une 404 avec les inconvénients que cela implique, où `find.where` renvoie une liste vide, ce qui m'arrange particulièrement bien.

Reply:
@Crash-- Update : c'est en fait dû à un problème dans le code, voir cette issue https://github.com/cozy/cozy-client/issues/301

Reply:
En gros une 404 génère ensuite inévitablement une typeError